spec/features/visitor_can_view_items_spec.rb

```diff
@@ -4,11 +4,13 @@
   feature "View items" do
     let!(:item_one) {Item.create(name: "Canon 7d",
                                  description: "The best camera ever (NOT)",
-                                 price: 15)}
+                                 price: 15,
+                                 image_url: 'http://www.bhphotovideo.com/images/images1000x1000/sony_sel1635z_vario_tessar_t_e_16_35mm_1082052.jpg')}
```

Comment:
Line is too long. [149/80]<br>Expression at 8, 149 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.